### ISSUE TITLE
DevTools Error: FAILED_DOCUMENT_REQUEST

### ISSUE THREAD

**Issue by kenbanks** [GoogleChrome/lighthouse#4776]
**Initial URL**: http://audio-mp3.ibiblio.org:8000/wncw-128k
**Chrome Version**: 67.0.3371.0
**Error Message**: FAILED_DOCUMENT_REQUEST
**Stack Trace**:
```
LHError: FAILED_DOCUMENT_REQUEST
    at Function.getPageLoadError (chrome-devtools://devtools/remote/serve_file/@497c718be40e845d85a70a9b3a01c6f1ebe44efc/audits2_worker/audits2_worker_module.js:917:27)
    at pass.then._ (chrome-devtools://devtools/remote/serve_file/@497c718be40e845d85a70a9b3a01c6f1ebe44efc/audits2_worker/audits2_worker_module.js:922:270)
```

**Comment by kenbanks** [comment]
I reported this same bug (as #4775), but I noticed the Audit was set for Mobile.  Here is the same failure with the Audit set for Desktop.

This station WNCW has had a working mp3 audio stream for many years. Their stream continues to work flawlessly in the Firefox and Eclipse browsers. It has also always worked with Google Chrome (my primary browser) until recently. Chrome now only plays the first 13 seconds of the stream intro.
